nucleus/dataset.py

```diff
@@ -86,21 +87,36 @@ def items(self) -> List[DatasetItem]:
         return self._client.get_dataset_items(self.id)
 
     @sanitize_string_args
-    def autotag_scores(self, autotag_name, for_scores_greater_than=0):
-        """Export the autotag scores above a threshold, largest scores first.
+    def autotag_items(self, autotag_name, for_scores_greater_than=0):
+        """For a given Autotag of this dataset, export its tagged items with scores above a threshold, largest scores first.
 
-        If you have pandas installed, you can create a pandas dataframe using
+        :return: dictionary of the form
+            {
+                'autotagItems': AutotagItem[],
+                'autotag': Autotag
+            }
+        See https://dashboard.nucleus.scale.com/nucleus/docs/api#export-autotag-items for more details on the return types.
```

Comment:
Document return type here so that you can do

help(dataset.autotag_items) and directly see the return format

Reply:
(I know this is annoying and we should auto-generate API docs from these docstrings, we are working towards that and not there yet though, sorry)